### PR TITLE
[Browser SDK] send user/company on init

### PR DESCRIPTION
**Pull request by roncohen**
The BrowserSDK needs to send user/company on init. Moving it out of the React SDK and into the Browser SDK.

### CODE REVIEW

File: packages/browser-sdk/src/client.ts [hunk 1]
```diff
@@ -148,6 +148,18 @@ export class BucketClient {
 
     await this.featuresClient.initialize();
 
+    if (this.context.user) {
+      this.user(this.context.user).catch((e) => {
```
Comment (pavkam) [inline packages/browser-sdk/src/client.ts:152]: I wonder why my lsp complains about the unawaited promises and yours doesnt. I usually add the special void keyword in front to make typescript understand I don't want to await.

Comment (roncohen) [reply packages/browser-sdk/src/client.ts]: agreed, pretty weird. I've learned from your comments to always add the `void`. These two are awaited though :)

Comment (pavkam) [reply packages/browser-sdk/src/client.ts]: The catch call returns another promise though

Comment (roncohen) [reply packages/browser-sdk/src/client.ts]: `catch` is enough AFAICT? : https://typescript-eslint.io/rules/no-floating-promises/

Comment (pavkam) [reply packages/browser-sdk/src/client.ts]: Interesting. I think I've configured my TS LSP to be stricter and that's why it won't let me be in such cases. But i agree, it's enough.

**Review by pavkam — APPROVED** [review]

**Pull request merged**